Create newsletter-june-2017

hillaryfraley opened the pull request.

hillaryfraley committed:
Create newsletter-june-2017

ericholscher commented on docs/blog/newsletter-june-2017:
```diff
@@ -0,0 +1,110 @@
+Static site generators have been a popular topic for some time, but there are a lot of moving parts that can be intimidating for new users. One Slack participant wondered how static site tools like Sphinx, Asciidoctor, restructured text (rST), Jekyll, and Markdown fit together--or don't. Here's the insight from our community:
+
+* Sphinx, Jekyll, and Asciidoctor are parsing engines; rST, Markdown, and AsciiDoc are source file formats; and Python and Ruby are programming languages. So you can write in rST to publish with Sphinx, which is written in Python. Or, you can write in Markdown to publish in Jekyll, which is written in Ruby.
+* In general, Sphinx parses rST, and Jekyll parses Markdown. However, some flavors of Sphinx parse Markdown too, like `Read the Docs <https://docs.readthedocs.io/en/latest/>`_.
```
RTD isn't a flavor of Sphinx -- it uses recommonmark, which is just a generic Sphinx extension that anyone can use: http://recommonmark.readthedocs.io/en/latest/

ericholscher commented on docs/blog/newsletter-june-2017:
```diff
@@ -0,0 +1,110 @@
+
+* Sphinx, Jekyll, and Asciidoctor are parsing engines; rST, Markdown, and AsciiDoc are source file formats; and Python and Ruby are programming languages. So you can write in rST to publish with Sphinx, which is written in Python. Or, you can write in Markdown to publish in Jekyll, which is written in Ruby.
+* In general, Sphinx parses rST, and Jekyll parses Markdown. However, some flavors of Sphinx parse Markdown too, like `Read the Docs <https://docs.readthedocs.io/en/latest/>`_.
+* There are also plugins out there to translate Markdown to rST, and to make rST work with Jekyll. An example is `recommonmark <http://recommonmark.readthedocs.io/en/latest/>`_.
```
This doesn't make rst work with Jekyll, it makes markdown work with Sphinx.

ericholscher commented on docs/blog/newsletter-june-2017:
```diff
@@ -0,0 +1,110 @@
+* Sphinx, Jekyll, and Asciidoctor are parsing engines; rST, Markdown, and AsciiDoc are source file formats; and Python and Ruby are programming languages. So you can write in rST to publish with Sphinx, which is written in Python. Or, you can write in Markdown to publish in Jekyll, which is written in Ruby.
+* In general, Sphinx parses rST, and Jekyll parses Markdown. However, some flavors of Sphinx parse Markdown too, like `Read the Docs <https://docs.readthedocs.io/en/latest/>`_.
+* There are also plugins out there to translate Markdown to rST, and to make rST work with Jekyll. An example is `recommonmark <http://recommonmark.readthedocs.io/en/latest/>`_.
+* You write in Markdown to publish with Jekyll, but Jekyll actually publishes the static site in HTML.
```
All SSG's generate HTML, as a general rule, so this is a bit tautological.

ericholscher commented on docs/blog/newsletter-june-2017:
```diff
@@ -0,0 +1,110 @@
+* In general, Sphinx parses rST, and Jekyll parses Markdown. However, some flavors of Sphinx parse Markdown too, like `Read the Docs <https://docs.readthedocs.io/en/latest/>`_.
+* There are also plugins out there to translate Markdown to rST, and to make rST work with Jekyll. An example is `recommonmark <http://recommonmark.readthedocs.io/en/latest/>`_.
+* You write in Markdown to publish with Jekyll, but Jekyll actually publishes the static site in HTML.
+
```
I might be able to cover this in a nicer way -- but I'd probably end up with a blog post at the end of it, so maybe I should write a blog post, but I likely won't have time to do it before we send the newsletter :)

hillaryfraley committed:
Update newsletter-june-2017

Add corrections and suggestions from @ericholscher
Add upcoming meetups in various cities
Corrected messy writing in WTD Meetup Framework section

hillaryfraley committed:
Update newsletter-june-2017

Removed duplicate heading in Screenshots section

hillaryfraley committed:
Update newsletter-june-2017

Added NYC and Austin meet ups
